test/browser/features/DVA_Driving_Licence.feature

```diff
@@ -226,6 +242,18 @@ Feature: DVA Driving licence CRI Error Validations
     When User clicks on continue
     Then I can see the valid to date error in the error summary as You cannot use an expired driving licence
     And I can see the Valid to date field error as You cannot use an expired driving licence
+    And I check the page Title Error: Enter your details exactly as they appear on your UK driving licence – Prove your identity – GOV.UK
     Examples:
       |DVADrivingLicenceSubject       |InvalidValidToDay|InvalidValidToMonth|InvalidValidToYear|
       |DrivingLicenceSubjectHappyBilly|         10      |     01            |         2010     |
+
+  @mock-api:dva-ConsentError @validation-regression @build @staging
+  Scenario Outline:  DVA Driving Licence error validation when DVA consent checkbox is unselected
+    Given User enters DVA data as a <DVADrivingLicenceSubject>
+    And DVA consent checkbox is unselected
+    When User clicks on continue
+    Then I can see the DVA consent error summary as You must give your consent to continue
+    And I can see the DVA consent error on the checkbox as Error: You must give your consent to continue
```

Comment:
Do we not also want to check the tab title here?

Reply:
Tab title validation added.